src/javascript/JContent/actions/searchAction/SearchDialog/AdvancedSearch/AdvancedSearch.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Sql2Input from './Sql2Input';
+import {connect} from 'react-redux';
+import {Trans} from 'react-i18next';
+import {DxContext} from '@jahia/react-material';
+import {compose} from '~/utils';
+import {cmGoto} from '~/JContent/JContent.redux';
+import JContentConstants from '~/JContent/JContent.constants';
+import styles from './AdvancedSearch.scss';
+import SearchLocation from '../SearchLocation';
+import {Typography} from '@jahia/moonstone';
+
+export class AdvancedSearch extends React.Component {
```

Comment:
Could we have this component split in two, with a container and a "pure" UI component?

Reply:
AdvancedSearch is part of another story, I did some changes in it, but I mainly moved the file to right place, in the following story it should be transformed into a functional component as anyway the logic will likely not be compatible with the rest of the UI